docs/home/howto/LOGGING.md

```diff
@@ -175,6 +175,18 @@ PUT _template/template_01
 ```
 After applying this policy every new index created under this one will apply to it. There is also possibility to apply policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
+## How to export Kibana reports to csv format
+
+Since v0.10 Epiphany provide posibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature.
+
+More details about the plugin and how to export reports:  
+https://opendistro.github.io/for-elasticsearch-docs/docs/kibana/reporting
```

Comment:
hyperlink?

Reply:
> Could you please make lines max 120 chars?

Sure, good point, I was following the standard from Elasticsearch section which is wrong so I also improved a little bit the section with elasticsearch.